README.md

```diff
@@ -63,7 +63,7 @@ Generate them like so:
 
 ```bash
 go get github.com/segmentio/terraform-docs
-terraform-docs md ./ | cat -s | ghead -n -1 > README.md
```

Comment:
This line didn't include the `ghead` bit just for giggles. 😉 Let's keep it UNIXy 'round these parts and preserve those end of file newlines, eh?

Reply:
If that means moving from `ghead` to `head`, so be it. Maybe it's my own bias but I assume most people are developing on Macs which don't come with `ghead` but it's the closest thing to our Linux tool cousin.

Reply:
Actually, scrap the above comment. Let's do something that works for both modern Linux and Mac flavors. Mind changing it to the following:

```bash
terraform-docs md ./ | cat -s | tail -r | tail -n +2 | tail -r > README.md
```

Reply:
It's a utility that does the same thing as unix `head`. The problem is you have to install it via brew. `tail` should be shipped standard on any mac unless I'm off my rocker.